frontend/packages/component-library/tsup.config.ts

```diff
@@ -3,11 +3,32 @@ import {postcssModules, sassPlugin} from 'esbuild-sass-plugin';
 import type {Options} from 'tsup';
 import {glob} from 'glob';
 import {resolve} from 'node:path';
+import {cp} from 'node:fs/promises';
 
 const entryPoints = glob.sync('./src/**/index.ts', {
   posix: true,
 });
 
+async function onSuccess() {
+  const sourceDir = './src/common/styles';
+  const destDir = './dist/styles';
```

Comment:
Still don't agree we should change the export paths for styles :(

Reply:
I've removed styles from this PR, we'll work on styles based on our conversation earlier in a separate PR